Packs/EDL/Integrations/EDL/EDL_test.py

```diff
@@ -195,8 +196,43 @@ def test_create_new_edl(self, mocker):
                         'iopqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwertyuiopyuioplkjhgfdsa' \
                         'zxqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwertyuiopqwert' \
                         'yuiopqwertyuiopqwertyuiopqwertyuiop", "indicator_type": "URL"}\n' \
-                        '{"value": "demisto.com", "indicator_type": "URL"}' \
+                        '{"value": "demisto.com", "indicator_type": "URL"}'
 
+    def test_create_new_edl_edge_cases(self, mocker, requests_mock):
```

Comment:
Please add a docstring in the style of `Summary, Given, When Then`.
It's best to document what are the **edge cases** that you are testing.